config/configgrpc/configgrpc.go

```diff
@@ -107,6 +127,11 @@ type KeepaliveServerParameters struct {
 	Timeout               time.Duration `mapstructure:"timeout"`
 }
 
+// NewDefaultKeepaliveServerParameters() creates a new KeepaliveServerParameters with any default values set
+func NewDefaultKeepaliveServerParameters() *KeepaliveServerParameters {
+	return &KeepaliveServerParameters{}
```

Comment:
We should set some default values based on https://pkg.go.dev/google.golang.org/grpc@v1.62.1/keepalive#ServerParameters